Tests/Resources/bin/runtests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "<?php exit(defined('HHVM_VERSION') ? 1 : 0);" > /tmp/ishhvm.php
+
+php /tmp/ishhvm.php
```

Comment:
can't you simply check for an environment variable to know if we are on a HHVM instance?